selfdrive/car/hyundai/interface.py

```diff
@@ -49,6 +49,10 @@ def _get_params(ret, candidate, fingerprint, car_fw, experimental_long, docs):
         if candidate not in CANFD_RADAR_SCC_CAR:
           ret.flags |= HyundaiFlags.CANFD_CAMERA_SCC.value
     else:
+      # detect HDA2 with CAN and CAN-FD definitions with ADAS Driving ECU
+      if hda2:
+        ret.flags |= HyundaiFlags.CAN_CANFD_HDA2.value
```

Comment:
Left a similar comment in the non-HDA2 PR, but let's not add another HDA 2 flag. It makes more sense to have one flag that describes if HDA2 or not, and one to describe if it's this mixed CAN and CAN FD platform